GCRCatalogs/alphaq.py

```diff
@@ -248,6 +273,12 @@ def _collect_native_quantities(name, obj):
         if catalog_version < StrictVersion('2.1.2'):
             self._quantity_modifiers.update({
                 'position_angle_true':     (lambda pos_angle: np.rad2deg(np.rad2deg(pos_angle)), 'morphology/positionAngle'), #I converted the units the wrong way, so a double conversion is required.
+                'ellipticity_1_true':       'morphology/totalEllipticity1'),
```

Comment:
These lines should be moved to under `if catalog_version < StrictVersion('3.0'):` around Line 270.

Reply:
That's right...